pom.xml

```diff
@@ -269,7 +269,12 @@
 			<dependency>
 				<groupId>org.apache.httpcomponents</groupId>
 				<artifactId>httpclient</artifactId>
-				<version>4.5.3</version>
+				<version>4.5.13</version>
```

Comment:
@marianbuenosayres can we take this version number from variable from parent POM. so in future version change will be easy.

Reply:
Makes sense. Will do the change for httpclient and httpcore